src/_layouts/default.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html lang="en-GB">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8"/>
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
+/>
```

Comment:
Is this Prettier?

Reply:
It may be.. I don't think we have any project rules for html...

Reply:
Yeah probably not - we do have a `.prettierignore` file but I can't remember what `**` means. Everything?